### ISSUE TITLE
Accessibility: Add h1 tag to all pages

### ISSUE THREAD

**Issue by pdpinch** [mitodl/odl-video-service#383]
According to accessibility guidelines, every page should have one (and only one) h1 tag. 

- [x] On video detail page, I recommend making the video title an h1 
- [x] On the collections page, "My Collections" should be an h1
- [x] On the collection page, the collection name should be the h1
